src/components/biblio/BiblioWorkflow.js

```diff
@@ -564,11 +639,126 @@ const BiblioWorkflow = () => {
       },
   ];
 
+  const indexingWorkflowColumns = [
+    {
+      headerName: 'Workflow Process',
+      field: 'section',
+      flex: 1,
+      cellStyle: { textAlign: 'left' },
+      headerClass: 'wft-bold-header wft-header-bg',
+      sortable: true,
+      filter: true,
+    },
+    {
+      headerName: 'Current Status',
+      field: 'workflow_tag',
+      flex: 1,
+      cellStyle: { textAlign: 'left' },
+      headerClass: 'wft-bold-header wft-header-bg',
+      cellRenderer: GeneralizedDropdownRenderer,
+      cellRendererParams: (params) => ({
+        value: params.value,
+        node: params.node,
+        colDef: params.colDef,
+        options: params.data.options,
+        validateFn: (val) => params.data.options.some(opt => opt.value === val),
+        errorMessage: 'INVALID ATP ID',
+        isDisabled: false,
+      }),
+      sortable: true,
+      filter: true,
+    },
+    {
+      headerName: 'Email',
+      field: 'email',
+      flex: 1,
+      cellStyle: { textAlign: 'left' },
+      headerClass: 'wft-bold-header wft-header-bg',
+      sortable: true,
+      filter: true,
+    },
+    {
+      headerName: 'Date Updated',
+      field: 'date_updated',
+      flex: 1,
+      cellStyle: { textAlign: 'left' },
+      headerClass: 'wft-bold-header wft-header-bg',
+      sortable: true,
+      filter: true,
+    },
+  ];
+
   const containerStyle = {
     display: 'flex',
     justifyContent: 'center',
   };
 
+  const onIndexingWorkflowCellValueChanged = useCallback(async (params) => {
+    if (params.column.getColId() !== 'workflow_tag') return;
+  
+    const { data, oldValue, newValue } = params;
+    if (newValue === oldValue) return;
+
+    try {
+      if (data.reference_workflow_tag_id) {
+        // Existing tag - PATCH
+        await patchWorkflowTag(
+          data.reference_workflow_tag_id, 
+          { workflow_tag_id: newValue }
+        );
+      } else {
+        // New tag - POST
+        await postWorkflowTag({
+          reference_curie: referenceCurie,
+          mod_abbreviation: accessLevel,
+          workflow_tag_id: newValue
+        });
+      }
+    
+      // Refresh data after successful update
+      fetchIndexingWorkflowOverview();
+    } catch (error) {
+      const errorMessage = `API error: ${error.message}`;
+      setApiErrorMessage(errorMessage);
+      setShowApiErrorModal(true);
+      console.error('Error updating workflow tag:', error);
+    }
+  }, [referenceCurie, accessLevel, accessToken, fetchIndexingWorkflowOverview]);
+
+  // Helper function for POST request
+  const postWorkflowTag = async (data) => {
```

Comment:
Oh, nice.  If you wanted to, we could make these higher level so that other places that update WorkflowTag use them, but I'm fine with them being a separate thing.  I think at this point there's only a couple of other places that do this in this code